### PR TITLE
ADD: Language and currency setting on first run

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/ezaej7cg5ufqaf2aew89tbh9k4

[download apk](http://filestorage.bluewallet.io:1488/1592187464.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/7h5q7jhxtmatmqymu3ajcc0gj0

[download apk](http://filestorage.bluewallet.io:1488/1592230217.apk)

File: package.json [hunk 1]
```diff
@@ -117,6 +117,7 @@
     "react-native-image-picker": "1.1.0",
     "react-native-level-fs": "3.0.1",
     "react-native-linear-gradient": "2.5.6",
+    "react-native-localize": "  1.4.0",
```
Comment (Overtorment) [inline package.json:120]: ok lets merge it, but plz, no more dependencies

**Comment by Overtorment** [comment]
circleci fails with 


```
 ● Localization › has all keys in all locales
 

 
    TypeError: Cannot read property 'list' of undefined
 

 
      110 |           if (typeof en[key1][key2] !== 'string') {
 
      111 |             for (const key3 of Object.keys(en[key1][key2])) {
 
    > 112 |               if (typeof locale[key1][key2][key3] === 'undefined') {
 
          |                                         ^
 
      113 |                 console.error('Missing: ' + lang + '.' + key1 + '.' + key2 + '.' + key3);
 
      114 |                 issues++;
 
      115 |               }
 
```

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/1fpedxmz4rda3yecx0c049a5dw

[download apk](http://filestorage.bluewallet.io:1488/1592242458.apk)

**Comment by marcosrdz** [comment]
> circleci fails with
> 
> ```
>  ● Localization › has all keys in all locales
>  
> 
>  
>     TypeError: Cannot read property 'list' of undefined
>  
> 
>  
>       110 |           if (typeof en[key1][key2] !== 'string') {
>  
>       111 |             for (const key3 of Object.keys(en[key1][key2])) {
>  
>     > 112 |               if (typeof locale[key1][key2][key3] === 'undefined') {
>  
>           |                                         ^
>  
>       113 |                 console.error('Missing: ' + lang + '.' + key1 + '.' + key2 + '.' + key3);
>  
>       114 |                 issues++;
>  
>       115 |               }
>  
> ```

fixed.

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/af65qmw9d2a8gb9k643r2gyqc0

[download apk](http://filestorage.bluewallet.io:1488/1592243480.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/3m4u1a2wcukzfkc09fkww08fgr

[download apk](http://filestorage.bluewallet.io:1488/1592325761.apk)

**Comment by marcosrdz** [comment]
@Overtorment don't be shy..merge this PR..dont be shy.

**Comment by Overtorment** [comment]
Let’s rerelease 543 first and then merge it

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/amutac1x77uf6rgznhd6hhqr0m

[download apk](http://filestorage.bluewallet.io:1488/1592392653.apk)

**Pull request merged**